docs/GLOSSARY.rst

```diff
@@ -0,0 +1,15 @@
+Glossary
+========
+
+See README.rst for high level information about how to navigate the full documentation.
+
+Glossary of Terms
+-----------------
+
+There are many terms that are highly technical and are not required to use the system, but only to administer it. This glossary is intented to orient users who may be confused about these terms and wish to learn more, but by no means is understanding these necessary for usage of the WSI DeID.
```

Comment:
I would shorten this:  `This glossary contains technical terms for those wishing to learn more; understanding these terms is not necessary for usage of the WSI DeID.`